CODEOWNERS

```diff
@@ -0,0 +1,82 @@
+# https://help.github.com/articles/about-codeowners/
+# Note that GitHub teams must have WRITE access to the repository.
+
+# Default owners for everything in the repo unless a later match takes
+# precedence.
+*           @digital-asset/daml-owners
+CODEOWNERS  @digital-asset/daml-owners # Only owners can change this file
+
+VERSION     @digital-asset/daml-approvers # Any approver can approve a release
+
+# Build / CI / environment
+/3rdparty/          @digital-asset/infrastructure
+/bazel_tools/       @digital-asset/infrastructure
+/build-scripts/     @digital-asset/infrastructure
+/ci/                @digital-asset/infrastructure
+/dev-env/           @digital-asset/infrastructure
+/hazel/             @digital-asset/infrastructure
+/nix/               @digital-asset/infrastructure
+/infra/             @digital-asset/infrastructure
+/pipeline/          @digital-asset/infrastructure
+/rules_daml/        @digital-asset/infrastructure
+/replacements/      @digital-asset/infrastructure
+BAZEL-JVM.md        @digital-asset/infrastructure
+BAZEL-haskell.md    @digital-asset/infrastructure
+BAZEL.md            @digital-asset/infrastructure
+BUILD               @digital-asset/infrastructure
+WORKSPACE           @digital-asset/infrastructure
+azure-pipelines.yml @digital-asset/infrastructure
+build.Jenkinsfile   @digital-asset/infrastructure
+build.ps1           @digital-asset/infrastructure
+build.sh            @digital-asset/infrastructure
+deps.bzl            @digital-asset/infrastructure
+dotfiles            @digital-asset/infrastructure
+
+# Language
+/compiler/          @digital-asset/language
+/daml-foundations/  @digital-asset/language
+/libs-haskell/      @digital-asset/language
+/ghc-lib/           @digital-asset/language
+
+# SDK
+/release/           @digital-asset/language @digital-asset/infrastructure
+/sdk/               @digital-asset/language
+/da-assistant/      @digital-asset/language
+/daml-assistant/    @digital-asset/language
+
+# Runtime
+/daml-lf/           @digital-asset/daml-lf
+/ledger/            @digital-asset/ledger
+/ledger-api/        @digital-asset/ledger
+
+# Ecosystems
+/language-support/          @digital-asset/java @digital-asset/scala @digital-asset/js
+/language-support/java/     @digital-asset/java
+/language-support/js/       @digital-asset/js
+/language-support/scala/    @digital-asset/scala
+/scala-protoc-plugins/      @digital-asset/scala @digital-asset/ledger
+
+# Tools
+/extractor/         @digital-asset/extractor
+/navigator/         @digital-asset/navigator
+/web-ide/           @digital-asset/language
+
+# Misc
+/docs/              @digital-asset/docs
+/notices-gen/       @digital-asset/approvers
+/oss-compliance/    @digital-asset/approvers
+
+CHANGELOG           @digital-asset/approvers
```

Comment:
I'm a bit concerned this is going to actively discourage people from keeping the CHANGELOG up-to-date.

Reply:
That's the wrong CHANGELOG, so no one should be keeping it up to date.

Reply:
Whether this is a deprecated changelog or not, if it were in use `@digital-asset/approvers` means anyone who has write access can approve it. What would you propose @garyverhaegen-da?

Reply:
There was no description of what "approvers" mean. If it means everyone, I have no objection.

@neil-da if it's the wrong file, should it be deleted?

Reply:
No, it's handy because it points people at the actual changelog.

Reply:
@garyverhaegen-da from the email: "Add a @digital-asset/daml-approvers team for everyone who should be able to approve things. I recommend this starts with all code-contributors in the DAML Language, DAML Runtime, and DAML-on-X teams. Other employees can also be added on a case-by-case basis."